### PR TITLE
docker tutorial for turn.auth

### PR DESCRIPTION
**Pull request by DoruM**
This tutorial should be extended to also include turn.proxy and so on. At the moment only turn authentication is done

### CODE REVIEW

File: coturn/coturn/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  gateway:
+    build: gateway
+    ports:
+      - "8000:8000"
+    hostname: "kaazing.example.com"
```
Comment (dpwspoon) [inline coturn/coturn/docker-compose.yml]: I believe the gateway opts will need an early access feature flag for this, correct?

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Each subdirectory contains a scenario listed below and provides instructions on
 * HTTP Proxy - example coming soon.
 * High Availability and Clustering - example coming soon.
 * KWIC ([Kaazing WebSocket Intercloud Connect](http://kaazing.com/kwic/)) - example coming soon  ![EE]
+* [TURN and Kaazing](coturn)
```
Comment (dpwspoon) [inline README.md]: I would recommend top level file is webrtc.  Subfolder is coturn.

File: coturn/coturn/docker-compose.yml (hunk 1 above)
Comment (DoruM) [inline coturn/coturn/docker-compose.yml]: I thought that it is enough to add `<feature.turn.rest>true</feature.turn.rest>` in the xml that is copied in the container (which I did).

Comment (dpwspoon) [inline coturn/coturn/docker-compose.yml]: I see the misunderstanding is due to something I missed in the original PR: https://github.com/kaazing/gateway/pull/674/files#r74501870

**Pull request closed without merging**